File: Assets/AltUnityTester/AltUnityDriver/AltUnity/AltUnityObjectAction.cs

public struct AltUnityObjectAction {

	public string Component;
	public string Method;
	// public Dictionary<string, string> parameters;
	public string Parameters;
    public string TypeOfParameters;

    public AltUnityObjectAction(string component="", string method="", string parameters="", string typeOfParameters="",string assembly="")
    {
        Component = component;
        Method = method;
        Parameters = parameters;
        TypeOfParameters = typeOfParameters;
        Assembly = assembly;
    }


    public string Assembly;
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings/java/src/main/java/ro/altom/altunitytester/AltUnityObjectAction.java
package ro.altom.altunitytester;


public class AltUnityObjectAction {
    public String component;
    public String method;
    public String parameters;
    public String typeOfParameters;
    public String assembly;

    public AltUnityObjectAction(String componentName, String methodName, String parametersNames, String typeOfParameters,String assemblyName) {
        component = componentName;
        method = methodName;
        parameters = parametersNames;
        this.typeOfParameters=typeOfParameters;
        assembly=assemblyName;




    }
}
